docs/using-gitbase/configuration.md

```diff
@@ -86,37 +86,35 @@ Help Options:
 
 [server command options]
           --db=                                        Database name (default: gitbase)
-      -d, --directories=                               Path where the git repositories are located
-                                                       (standard and siva), multiple directories can
-                                                       be defined. Accepts globs.
-          --depth=                                     load repositories looking at less than
-                                                       <depth> nested subdirectories. (default: 1000)
-          --host=                                      Host where the server is going to listen
-                                                       (default: localhost)
-      -p, --port=                                      Port where the server is going to listen
-                                                       (default: 3306)
+      -d, --directories=                               Path where standard git repositories are located,
+                                                       multiple directories can be defined.
+          --format=[git|siva]                          Library format (default: git)
+          --bucket=                                    Bucketing level to use with siva libraries (default: 2)
```

Comment:
do we really need it? If yes, I would suggest to have at least 2 (so you can't set up less than 2).

Reply:
I don't think we should impose any limits. We are doing this so we can also read old sivas, for example from PGA. A use case can be copying some siva files in a directory without bucketing so `bucket=0` should be available. The default parameters for siva are compatible with the new download.

Reply:
but it's the same if I say `bucket=1234567879`

Reply:
I understand adding a maximum and probably not letting negative numbers that can crash the program. But I don't find useful setting the minimum bucketing level to 2.